At line 24, the function 'addch' if defined.
(32: addchnstr, 40:addchstr).
Manual experimentation shows that renaming `addch` to e.g. `addchXXX` will fix the problem.
This means that the preprocessor is somehow replacing addch with another function.

(required for static build)

This fixes the following error:
ncurses/6.2: ../ncurses/lib_gen.c(24): error C2059: syntax error: '('
ncurses/6.2: ../ncurses/lib_gen.c(24): error C2143: syntax error: missing ')' before 'const'
ncurses/6.2: ../ncurses/lib_gen.c(24): error C2143: syntax error: missing '{' before 'const'
ncurses/6.2: ../ncurses/lib_gen.c(24): error C2059: syntax error: ')'
ncurses/6.2: ../ncurses/lib_gen.c(25): error C2054: expected '(' to follow 'z'
ncurses/6.2: ../ncurses/lib_gen.c(32): error C2059: syntax error: '('
ncurses/6.2: ../ncurses/lib_gen.c(32): error C2143: syntax error: missing ')' before 'const'
ncurses/6.2: ../ncurses/lib_gen.c(32): error C2143: syntax error: missing '{' before 'const'
ncurses/6.2: ../ncurses/lib_gen.c(32): error C2059: syntax error: ')'
ncurses/6.2: ../ncurses/lib_gen.c(32): error C2059: syntax error: '<parameter-list>'
ncurses/6.2: ../ncurses/lib_gen.c(40): error C2059: syntax error: '('
ncurses/6.2: ../ncurses/lib_gen.c(40): error C2143: syntax error: missing ')' before 'const'
ncurses/6.2: ../ncurses/lib_gen.c(40): error C2143: syntax error: missing '{' before 'const'
ncurses/6.2: ../ncurses/lib_gen.c(40): error C2059: syntax error: ')'
ncurses/6.2: ../ncurses/lib_gen.c(40): error C2373: 'z': redefinition; different type modifiers       ncurses/6.2: ../ncurses/lib_gen.c(24): note: see declaration of 'z'
ncurses/6.2: ../ncurses/lib_gen.c(40): error C2059: syntax error: '-'
ncurses/6.2: ../ncurses/lib_gen.c(48): error C2059: syntax error: '('
ncurses/6.2: ../ncurses/lib_gen.c(48): error C2143: syntax error: missing ')' before 'const'
ncurses/6.2: ../ncurses/lib_gen.c(48): error C2143: syntax error: missing '{' before 'const'
ncurses/6.2: ../ncurses/lib_gen.c(48): error C2059: syntax error: ')'
ncurses/6.2: ../ncurses/lib_gen.c(48): error C2371: 'a1': redefinition; different basic types
ncurses/6.2: ../ncurses/lib_gen.c(32): note: see declaration of 'a1'
ncurses/6.2: ../ncurses/lib_gen.c(48): error C2059: syntax error: '<parameter-list>'
ncurses/6.2: ../ncurses/lib_gen.c(56): error C2059: syntax error: '('
ncurses/6.2: ../ncurses/lib_gen.c(56): error C2143: syntax error: missing ')' before 'const'
ncurses/6.2: ../ncurses/lib_gen.c(56): error C2143: syntax error: missing '{' before 'const'
ncurses/6.2: ../ncurses/lib_gen.c(56): error C2059: syntax error: ')'
ncurses/6.2: ../ncurses/lib_gen.c(56): error C2373: 'z': redefinition; different type modifiers       ncurses/6.2: ../ncurses/lib_gen.c(24): note: see declaration of 'z'
ncurses/6.2: ../ncurses/lib_gen.c(56): error C2059: syntax error: '-'
ncurses/6.2: ../ncurses/lib_gen.c(64): error C2059: syntax error: '('
ncurses/6.2: ../ncurses/lib_gen.c(64): error C2146: syntax error: missing ')' before identifier 'attr_t'
ncurses/6.2: ../ncurses/lib_gen.c(64): error C2059: syntax error: ')'
ncurses/6.2: ../ncurses/lib_gen.c(64): error C2059: syntax error: '<parameter-list>'
ncurses/6.2: ../ncurses/lib_gen.c(64): fatal error C1903: unable to recover from previous error(s); stopping compilation
ncurses/6.2: Internal Compiler Error in C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64\cl.exe.  You will be prompted to send an error report to Microsoft later.
ncurses/6.2: INTERNAL COMPILER ERROR in 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64\cl.exe'
ncurses/6.2:     Please choose the Technical Support command on the Visual C++
ncurses/6.2:     Help menu, or open the Technical Support help file for more information
--- ncurses/base/MKlib_gen.sh
+++ ncurses/base/MKlib_gen.sh
@@ -432,7 +432,7 @@
 		print "#define NCURSES_ATTR_T int"
 		print "#include <ncurses_cfg.h>"
 		print ""
-		print "#undef NCURSES_NOMACROS	/* _this_ file uses macros */"
+		print "#define NCURSES_NOMACROS	/* _this_ file uses macros */"
 		print ""
 		print "#include <curses.priv.h>"
 		print ""
